otelcol/moved.go

```diff
@@ -62,8 +68,23 @@ type ConfigProviderSettings = service.ConfigProviderSettings //nolint:staticchec
 // * Then unmarshalls the confmap.Conf into the service Config.
 var NewConfigProvider = service.NewConfigProvider //nolint:staticcheck
 
-// NewCommand constructs a new cobra.Command using the given CollectorSettings.
-var NewCommand = service.NewCommand //nolint:staticcheck
-
 // Config defines the configuration for the various elements of collector or agent.
 type Config = service.Config //nolint:staticcheck
+
+func newDefaultConfigProviderSettings(uris []string) ConfigProviderSettings {
```

Comment:
where are the funcs in this file coming from? I'm not seeing this in the PR

Reply:
They are copied from service (still used there as well), I will remove them when I will be able to move more of the deprecated code.